web-registry/src/components/organisms/ProjectCardsSection/ProjectCardsSection.tsx

```diff
@@ -67,7 +69,7 @@ export function ProjectCardsSection({
                   onButtonClick && (() => onButtonClick({ project }))
                 }
                 purchaseInfo={project.purchaseInfo}
-                href={`/projects/${project.id}`}
+                onClick={() => navigate(`/projects/${project.id}`)}
```

Comment:
I guess we should use `linkComponent` along with the href as we do elsewhere in the app instead of calling `navigate`.
No need to change it here, probably out of the scope of this PR.

Reply:
Why this change @victordelval ? 

In the `ProjectDetails` case, this causes the navigation to change, but the new project data doesn't load. I believe this was a bug before.

Reply:
@erikalogie @mhagel @flagrede Oh I see! That's what Erika told me yesterday but I wasn't realizing it! Thanks for insisting.

I had changed that, prior confirmation with Erika, to unify the click behavior with the page of all projects, where the cards are clickable to navigate, on its entire surface, not just on the photo..

I guess the issue is that the data fetching in this page is not yet refactored. I'm going back to the original approach with `href` property in the meantime.